src/server/ApiManager.js

```diff
@@ -0,0 +1,61 @@
+class ApiManager {
+
+    constructor() {
+
+        this.apiUrlEndpoint = 'https://campmor.ampify.wompmobile.com/campmor';
+        this.apiCategoriesEndpoint = this.apiUrlEndpoint + '/fetchCategories';
+        this.apiProductEndpoint = this.apiUrlEndpoint + '/fetchProduct';
+
+        this.apiUrlMap = new Map();
+
+        this.apiUrlMap.set('men-shirts', '200368502');
+        this.apiUrlMap.set('men-shorts', '200368503');
+        this.apiUrlMap.set('women-shirts', '200368507');
+        this.apiUrlMap.set('women-shorts', '200368509');
+
```

Comment:
I've never gotten to use a real map in JS!  So I applaud your trying it out.  But I feel like this is verbose, and it will get way more so as we have more categories and mappings to do.  Would you consider one of the following:

1. Creating this map from an array?  e.g.,

```javascript
var a = [['a', 1], ['b', 2], ['c', 3]];
var map = new Map(a);
```

2. Simply using an old fashioned an object here instead?  It's not as cool as a Map, but it also has a shorter syntax for access...

Reply:
Hi Ben, thanks for the feedback.
I transformed the initialization of the Map into what you suggested in option #1, and submitted the change.
Please, let me know your thoughts.